stock_cycle_count/models/stock_inventory.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2017-2022 ForgeFlow S.L.
+#   (http://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+PERCENT = 100.0
+
+
+class StockInventory(models.Model):
+    _inherit = "stock.inventory"
+
+    @api.depends("state", "stock_quant_ids")
+    def _compute_inventory_accuracy(self):
+        for inv in self:
+            theoretical = sum(inv.stock_quant_ids.mapped(lambda x: abs(x.quantity)))
+            abs_discrepancy = sum(
+                inv.stock_quant_ids.mapped(lambda x: abs(x.discrepancy_qty))
```

Comment:
```suggestion
                inv.stock_quant_ids.mapped(lambda x: abs(x.inventory_diff_quantity))
```
"discrepancy_qty" it seems that is no longer available in version 16, i suggest change this attribute for "inventory_diff_quantity"